File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    answer = ''
    count = 0
    idx = []
    alpa='abcdefghijklmnopqrstuvwxyz'
    while age>0 :
        idx.insert(0, age%10)
        age = age // 10
    for i in idx:
        answer += alpa[i]
    return answer

File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        String alpa ="abcdefghijklmnopqrstuvwxyz";
        String[] arr = alpa.split("");
        while(age>0){
            int i = age%10;
            age /= 10;
            answer += arr[i];
        }
        StringBuffer sb = new StringBuffer(answer);
        answer = sb.reverse().toString();
        return answer;
    }
}